src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,138 @@
+package duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Storage {
+
+    // initializing saver to save todoList tasks to relativePath
+    private String fileSeparator = System.getProperty("file.separator");
+    private String relativePath = "data" + fileSeparator + "duke.txt";
+
+    /**
+     * Constructor of Storage object.
+     *
+     * @throws IOException
+     */
+    public Storage() throws IOException {
+
+        File f = new File(this.relativePath);
+        // if file does not exist, create file based on filepath
+        if (!f.exists()) {
+            // create data directory
+            Files.createDirectories(Path.of(f.getParent()));
+
+            // create new file
+            if (f.createNewFile()) {
+                System.out.println(this.relativePath + " file created in the project root directory");
```

Comment:
Perhaps length of line could be shorter (e.g line break before operator) to abide by Java Coding Standard 👍

Reply:
Yup I agree, would be good to break and let the '+' be at the front of the new line